GitExtensions/Project.Loc.targets

```
@@ -68,7 +68,7 @@
           ContinueOnError="false" />
 
     <!-- 5. Verify no changes to English*.xlf -->
-    <Exec Command="git add *.xlf --ignore-errors"
+    <Exec Command="cmd /c git add *.xlf --ignore-errors > /dev/null 2>&1"
```

The change hides other errors too, suggest to suppress autocrlf instead.
(Have not tried this in a build.)
```suggestion
    <Exec Command="git -c core.autocrlf=false add *.xlf --ignore-errors"
```